### PR TITLE
Update to Master for fixes.

### PR DESCRIPTION
**Pull request by flammableliquids**
Read comments for full details.
Added functionality to curtail number of concurrent handbrake jobs.
Added a logging fix to not pump to the same log file.
Updated readme instructions to be a little more functional, and also to enable nvenc support, plus refer to the correct repo.

### CODE REVIEW

**Comment by colonelpopcorn** [comment]
Fix conflicts in readme?

**Comment by flammableliquids** [comment]
> 
> 
> Fix conflicts in readme?

Fixed this now.

File: arm/ripper/handbrake.py [hunk 1]
```diff
@@ -26,6 +29,22 @@ def handbrake_mainfeature(srcpath, basepath, logfile, job):
...
+        #Count how many times handbrake is currently running
+        #procs = subprocess.check_output(['ps', '-a', '-c', '-ocomm=']).splitlines()
+        #TRANSCODE-NUMBER = procs.count('HandBrakeCLI')
+        transcode_number = sum(1 for proc in psutil.process_iter() if proc.name() == 'HandBrakeCLI')
+        logging.info("Number of HandBrakeCLI Processes running is:" + str(transcode_number) + " going to waiting 12 seconds.")
+        time.sleep(12)
+    
+    logging.debug("Setting job status to 'transcoding'")
+    job.status = "transcoding"
+
```
Comment (colonelpopcorn) [inline arm/ripper/handbrake.py:47]: Could this be extracted into a method/function? It just seems like a code smell to me that there's a duplicate here.

File: arm/ripper/handbrake.py [hunk 2]
```diff
@@ -94,6 +113,19 @@ def handbrake_all(srcpath, basepath, logfile, job):
...
+    transcode_number = 9999999
+    while transcode_number >= job.config.MAX_CONCURRENT_TRANSCODES:
+        #Count how many times handbrake is currently running
+        #procs = subprocess.check_output(['ps', '-a', '-c', '-ocomm=']).splitlines()
+        #TRANSCODE-NUMBER = procs.count('HandBrakeCLI')
+        transcode_number = sum(1 for proc in psutil.process_iter() if proc.name() == 'HandBrakeCLI')
+        logging.info("Number of HandBrakeCLI Processes running is:" + str(transcode_number) + " going to waiting 12 seconds.")
+        time.sleep(12)
+    
+    job.status = "transcoding"
```
Comment (colonelpopcorn) [inline arm/ripper/handbrake.py:128]: See my comment above.

File: arm/ripper/logger.py [hunk 3]
```diff
@@ -23,9 +23,22 @@ def setuplogging(job):
...
+        ilogfull = cfg['LOGPATH'] + logfile
+        if os.path.isfile(ilogfull):
+            logfile = str(job.label) + "_" + str(round(time.time() * 100)) + ".log"
+            logfull = cfg['LOGPATH'] + logfile
+        else:
+            logfull = cfg['LOGPATH'] + logfile       
     else:
-        logfull = cfg['LOGPATH'] + "/" + logfile
+        #Check to see if file already exists, if so, create a new file
+        iilogfull = cfg['LOGPATH'] + "/" + logfile
```
Comment (colonelpopcorn) [inline arm/ripper/logger.py:35]: More verbosity in variable names would be good, I don't understand what iilogfull means.

File: arm/ripper/logger.py [hunk 4]
```diff
@@ -23,9 +23,22 @@ def setuplogging(job):
         logfile = job.label + ".log"
 
     if cfg['LOGPATH'][-1:] == "/":
-        logfull = cfg['LOGPATH'] + logfile
+        #Check to see if file already exists, if so, create a new file
+        ilogfull = cfg['LOGPATH'] + logfile
```
Comment (colonelpopcorn) [inline arm/ripper/logger.py:27]: More verbosity in variable names would be good, I don't understand what ilogfull means.

File: arm/ripper/utils.py [hunk 5]
```diff
@@ -134,6 +134,15 @@ def rename_files(oldpath, job):
...
     logging.info("Changing directory name from " + oldpath + " to " + newpath)
 
+    # Sometimes a job fails, after the rip, but before move of the tracks into the folder, at which point the below command
+    # will move the newly ripped folder inside the old correctly named folder.
+    # This can be a problem as the job when it tries to move the files, won't find them.
+    # other than putting in an error message, I'm not sure how to perminently fix this problem.
+    # Maybe I could add a configurable option for deletion of crashed job files?
+
+    if os.path.isdir(newpath):
+        logging.info("Error: The 'new' directory already exists, ARM will probably copy the newly ripped folder into the old-new folder.")
```
Comment (colonelpopcorn) [inline arm/ripper/utils.py:144]: Could be good to open an issue for this, just to keep track of it.

**Review by colonelpopcorn — CHANGES_REQUESTED** [review]
Refactoring to a function in `handbrake.py` is just a suggestion, but it would be nice if you could change the `ilogfull` and `iilogfull` names to something more meaningful in `logger.py`

**Comment by flammableliquids** [comment]
Closing, will modify and re-submit

**Comment by flammableliquids** [comment]
@colonelpopcorn please review #366

**Pull request closed without merging**